src/push-build-information.ts

```diff
@@ -43,7 +57,8 @@ export async function pushBuildInformation(
       Branch: branch,
       VcsType: 'Git',
       VcsRoot: `${repoUri}`,
-      VcsCommitNumber: context.sha
+      VcsCommitNumber: context.sha,
+      Commits: commits
```

Comment:
Is it okay that `commits` here could possibly be `undefined` or should we be defaulting to an empty array or null instead when we initialise the variable on line 36?

Reply:
The .NET deserialization _should_ handle this, **but** for safety I will update it to be an empty array for anything other than `push` events